Assets/HoloToolkit/Input/Scripts/Focus/FocusManager.cs

```diff
@@ -696,6 +755,23 @@ private float GetPointingExtent(PointerData pointer)
             return null;
         }
 
+        /// <summary>
+        /// Helper for assigning world space canvases event cameras.
+        /// <remarks>Can be used at runtime.</remarks>
+        /// </summary>
+        public void UpdateCanvasEventSystems()
```

Comment:
The difference with the previous implementation is that you actually had register all your canvases with the `FocusManager` which I didn't like.  This is different because it searches the scene for you, and then assigns the canvases that only have their render mode set to `RenderMode.WorldSpace`.

Also, this is never run by the MRTK at runtime, I only added this here as a helper utility method for people who are generating canvases at runtime, and when you add the input system to your scene via the scene setup wizard.

If you don't want to assign the canvas then don't assign the uiRaycast camera to your canvas' event camera.

Reply:
@keveleigh, I added the ability to choose if you don't want the canvases to have the UIRaycastCamera as their world camera.

Reply:
Also, btw, if you don't want to use the RaycastCamera, the MainCamera is the default when the canvas' worldCamera is null anyway, so the only thing you'll need to do is assign the main camera in the Focus Manager. But again, it's **_highly_** recommended you use the RaycastCamera on the EventSystem.